apps/frontend/src/libs/components/input/input.tsx

```diff
@@ -19,9 +19,12 @@ type Properties<T extends FieldValues> = {
 	name: FieldPath<T>;
 	placeholder?: string;
 	rightIcon?: JSX.Element;
+	rows?: number;
 	type?: "email" | "password" | "text";
 };
 
+const MIN_ROWS_TO_BE_TEXTAREA = 2;
```

Comment:
maybe it should be in separate file? like constants

Reply:
I just didn't want to create that folders structure `libs/constants/...` just for this little constant... But your suggestion sounds right

Reply:
I think we don't need this constant at all. In case we pass any value to rows, we will render textarea, even if it's 1

Reply:
Sure